example.py

```diff
@@ -124,11 +130,11 @@ def main():
         logging.fatal(e)
         return
     create_accounts(conn)
-    print_balances(conn)
+    ids = print_balances(conn)
```

Comment:
I think it makes more sense to return ids from `create_accounts()` rather than `print_balances()`

Reply:
Done.